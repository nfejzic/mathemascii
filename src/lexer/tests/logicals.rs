use super::Snapshot;

super::test_snap!(and_or_not, "andornot and or not");
super::test_snap!(implications, "implies <=> => iff");
super::test_snap!(quantors, "AA EE forallexists AAEE");
super::test_snap!(other, "_|_ bottop TT |-- |==");
