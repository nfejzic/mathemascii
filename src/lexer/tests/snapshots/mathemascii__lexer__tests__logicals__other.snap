---
source: src/lexer/tests/logicals.rs
expression: "Snapshot((input, tokens))"
---
_|_ bottop TT |-- |==

_|_
^^^ -> Logical(Bottom) at: 0 -> 3

    bot
    ^^^ -> Logical(Bottom) at: 4 -> 7

       top
       ^^^ -> Logical(Top) at: 7 -> 10

           TT
           ^^ -> Logical(Top) at: 11 -> 13

              |--
              ^^^ -> Logical(VerticalDash) at: 14 -> 17

                  |==
                  ^^^ -> Logical(Models) at: 18 -> 21


